public/template/messages.jade

```diff
@@ -10,9 +10,9 @@
       if _.isObject(activity)
         li
           small!= activity.content
-
-    if messages.items.length > 0
-      li.divider
-
+    if !messages.items.length
```

Comment:
Is the same but more short and if returns 'undefined' for some reason also works